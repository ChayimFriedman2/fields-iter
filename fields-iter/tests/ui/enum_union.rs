use fields_iter::FieldsInspect;

#[derive(FieldsInspect)]
enum Foo {}

#[derive(FieldsInspect)]
union Bar {
    _foo: (),
}

fn main() {}
