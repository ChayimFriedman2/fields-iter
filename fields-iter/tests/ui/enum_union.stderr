error: only structs are supported for `#[derive(FieldsInspect)]`
 --> tests/ui/enum_union.rs:4:1
  |
4 | enum Foo {}
  | ^^^^^^^^^^^

error: only structs are supported for `#[derive(FieldsInspect)]`
 --> tests/ui/enum_union.rs:7:1
  |
7 | / union Bar {
8 | |     _foo: (),
9 | | }
  | |_^
